### ISSUE TITLE
[py3] dipy.io.tests.test_dpy.test_dpy: TypeError: Array objects cannot currently deal with void, unicode or object arrays

### ISSUE THREAD

**Issue by ignatenkobrain** [nipy/dipy#756]
``` python
======================================================================
ERROR: dipy.io.tests.test_dpy.test_dpy
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/lib64/python3.4/site-packages/numpy/testing/decorators.py", line 146, in skipper_func
    return f(*args, **kwargs)
  File "/builddir/build/BUILD/python3-python-dipy-0.9.2-1.fc24/build/lib.linux-x86_64-3.4/dipy/io/tests/test_dpy.py", line 23, in test_dpy
    dpw = Dpy(fname, 'w')
  File "/builddir/build/BUILD/python3-python-dipy-0.9.2-1.fc24/build/lib.linux-x86_64-3.4/dipy/io/dpy.py", line 69, in __init__
    self.version=self.f.createArray(self.f.root,'version',['0.0.1'],'Dpy Version Number')
  File "/usr/lib64/python3.4/site-packages/tables/_past.py", line 35, in oldfunc
    return obj(*args, **kwargs)
  File "/usr/lib64/python3.4/site-packages/tables/file.py", line 1152, in create_array
    obj=obj, title=title, byteorder=byteorder)
  File "/usr/lib64/python3.4/site-packages/tables/array.py", line 187, in __init__
    byteorder, _log)
  File "/usr/lib64/python3.4/site-packages/tables/leaf.py", line 262, in __init__
    super(Leaf, self).__init__(parentnode, name, _log)
  File "/usr/lib64/python3.4/site-packages/tables/node.py", line 267, in __init__
    self._v_objectid = self._g_create()
  File "/usr/lib64/python3.4/site-packages/tables/array.py", line 204, in _g_create
    raise TypeError("Array objects cannot currently deal with void, "
TypeError: Array objects cannot currently deal with void, unicode or object arrays
```

**Comment by arokem** [comment]
Could you try this on current `master`? Do you get that there as well?

We're not seeing this on our CI on 3.4

**Comment by ignatenkobrain** [comment]
@arokem tried last master. The same error.

```
$ rpm -q python3-tables python3 python3-numpy
python3-tables-3.2.0-3.fc23.x86_64
python3-3.4.3-5.fc24.x86_64
python3-numpy-1.10.1-1.fc24.x86_64
```

**Comment by skoudoro** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [#756](https://trello.com/c/EzR6eeRh/10-756)

**Comment by skoudoro** [comment]
I suppose pytables version was the problem. Pytables API change a lot between version 2 and 3. It should be fix via #1147. 
So can you try it again in your environment @ignatenkobrain ?  Should I close this issue @arokem  ?

**Comment by arokem** [comment]
Yes - let's close this for now. If this is still a thing, please feel free to reopen.
